### PR TITLE
Update readme with EPTQ

### PR DESCRIPTION
**Pull request by ofirgo**
Added a section regarding EPTQ with a reference to the arXive paper to the repository README.
Also, Added GPTQ README that gives a short explanation of the feature and highlights the EPTQ algorithm.

### CODE REVIEW

File: model_compression_toolkit/gptq/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Gradient-based Post-Training Quantization (GPTQ)
+
+The GPTQ package allows to run advanced quantization algorithm, in which the quantized parameters rounding is optimized after the initial quantization is applied.
```
Comment (lior-dikstein) [inline model_compression_toolkit/gptq/README.md:3]: The English here is not correct. This is an example of one way to revise it:

GPTQ is a core feature of MCT, offering an advanced quantization algorithm that optimizes parameter rounding after the initial quantization process. It preserves the principles of post-training quantization, utilizing a small, unlabeled dataset without the need for additional model information.

File: model_compression_toolkit/gptq/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+GPTQ preserves the principals of post-training quantization - it uses a small. unlabeled dataset, and does not require any additional information regarding the model.
+
+MCT provides several symmetric and uniform [trainable quantizers](../trainable_infrastructure/README.md) that allow to retrain the model. 
+
```
Comment (lior-dikstein) [inline model_compression_toolkit/gptq/README.md:8]: MCT provides a variety of symmetric and uniform [trainable quantizers](../trainable_infrastructure/README.md) that allow you to retrain your model efficiently.

File: model_compression_toolkit/gptq/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## Enhanced Post-Training Quantization (EPTQ)
+
+EPTQ is an in-house developed gradient-based quantization scheme that allows to retrain the quantized parameters [1].
+It is implemented for both Keras and PyTorch frameworks, and can be utilized via the GPTQ API.
+EPTQ is the default algorithm used when running GPTQ.
+
+EPTQ utilizes Hessian information (Hessian matrix trace w.r.t. pre-defined layer's activation tensors) to construct an optimization objective that gives special attention to the layers that are more sensitive to quantization.
```
Comment (lior-dikstein) [inline model_compression_toolkit/gptq/README.md:15]: EPTQ leverages Hessian information (Hessian matrix trace with respect to pre-defined layer activation tensors) to construct an optimization objective that prioritizes layers more sensitive to quantization.

File: model_compression_toolkit/gptq/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+The Hessian information is approximated in a label-free manner, thus, does not require any additional data beside the unlabeled data provided to the original PTQ method. 
+
+<img src="../../docsrc/images/eptq_overview.svg" width="10000">
+
+
+## GPTQ Usage
+
+For an example of how to use GPTQ in MCT with TensorFlow or PyTorch on various models and tasks, check out the [tutorials package](../../tutorials),
+where you can find extended explanation and notebook examples for running GPTQ, along with a [quick-start](../../tutorials/quick_start/README.md) framework for simple execution.
+
```
Comment (lior-dikstein) [inline model_compression_toolkit/gptq/README.md:25]: For detailed examples and tutorials on using GPTQ in MCT with TensorFlow or PyTorch across various models and tasks, please refer to the [tutorials package](../../tutorials). You will find comprehensive explanations, notebook examples, and a [quick-start guide](../../tutorials/quick_start/README.md) for straightforward execution.

**Review by lior-dikstein — APPROVED** [review]

File: model_compression_toolkit/gptq/README.md (hunk 1 above)
Comment (ofirgo) [reply model_compression_toolkit/gptq/README.md:3]: That's why I asked you to review this PR, thanks (;

**Pull request merged**